### PR TITLE
37: Add download section for the latest 3.7 release

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
This patch updates the downloads page of the website to mention the latest release and link to the download artifacts

### CODE REVIEW

**Comment by stanislavkozlovski** [comment]
so far these links don't exist and i haven't verified any of them, since the release is not done.

Not sure what the actual process is - whether we:
1) pre-approve, and I change the links & merge once finished releasing
2) wait for the release, update this, and then get an approval and merge

I would assume 1) is more flexible and preferable, since it allows me to update the website right after the release (avoiding any inconsistency). But I assume also we have time between the actual release and the announcement.

File: downloads.html [hunk 1]
```diff
@@ -6,12 +6,45 @@
...
     <p>
     You can verify your download by following these <a href="https://www.apache.org/info/verification.html">procedures</a> and using these <a href="https://downloads.apache.org/kafka/KEYS">KEYS</a>.
     </p>
 
+
+    <span id="3.7.0"></span>
+    <h3 class="download-version">3.7.0<a href="#3.7.0"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+    <ul>
+        <li>
+            Released Feb 21, 2024
```
Comment (stanislavkozlovski) [inline downloads.html]: This will change to the date of the **release** announcement

File: downloads.html [hunk 2]
```diff
@@ -6,12 +6,45 @@
...
+        </li>
+        <li>
+            <a href="https://downloads.apache.org/kafka/3.7.0/RELEASE_NOTES.html">Release Notes</a>
+        </li>
+        <li>
+            Docker image: <a href="https://hub.docker.com/layers/apache/kafka/3.7.0-rc4/images/sha256-3e324d2bd331570676436b24f625e5dcf1facdfbd62efcffabc6b69b1abc13cc?context=explore">apache/kafka:3.7.0</a>.
```
Comment (stanislavkozlovski) [inline downloads.html]: this needs to get updated 100%, since I don't have the image hash yet

File: downloads.html [hunk 3]
```diff
@@ -6,12 +6,45 @@
...
+                <li>Scala 2.13 &nbsp;- <a href="https://downloads.apache.org/kafka/3.7.0/kafka_2.13-3.7.0.tgz">kafka_2.13-3.7.0.tgz</a> (<a href="https://downloads.apache.org/kafka/3.7.0/kafka_2.13-3.7.0.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/3.7.0/kafka_2.13-3.7.0.tgz.sha512">sha512</a>)</li>
+            </ul>
+            We build for multiple versions of Scala. This only matters if you are using Scala and you want a version
+            built for the same Scala version you use. Otherwise any version should work (2.13 is recommended).
+        </li>
+    </ul>
+
+    <p>
+        Kafka 3.7.0 includes a significant number of new features and fixes.
+        For more information, please read our <a href="https://kafka.apache.org/blog#apache_kafka_360_release_announcement" target="_blank">blog post</a>
```
Comment (divijvaidya) [inline downloads.html]: typo: this should point to 370 release blog

**Comment by divijvaidya** [comment]
> But I assume also we have time between the actual release and the announcement.

Yes, we can publish this PR *after* uploading artifacts to maven etc.

File: downloads.html (hunk 3 above)
Comment (stanislavkozlovski) [inline downloads.html]:
```suggestion
        For more information, please read our <a href="https://kafka.apache.org/blog#apache_kafka_370_release_announcement" target="_blank">blog post</a>
```

**Review by divijvaidya — APPROVED** [review]
Conditional approval assuming we would update the date and docker image link after uploading the artifacts. Please feel free to merge after 37 artifacts are uploaded and you have updated the date/dicker image link, without waiting for another approval.

**Comment by stanislavkozlovski** [comment]
updated the docker image and the release date, to today

**Pull request merged**